db/migrate/20230119163631_add_post_ref_to_likes.rb

```diff
@@ -0,0 +1,5 @@
+class AddPostRefToLikes < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :post, null: false, foreign_key: true
```

Comment:
- [ ]  @topeogunleye kindly check your reference it should be the post and not the likes since its referencing the post